tests/foreman/ui/test_computeresource_gce.py

```diff
@@ -73,9 +77,7 @@ def test_positive_default_end_to_end_with_custom_profile(
                 'description': cr_description,
                 'provider': FOREMAN_PROVIDERS['google'],
                 'provider_content.http_proxy.value': http_proxy.name,
-                'provider_content.google_project_id': gce_cert['project_id'],
-                'provider_content.client_email': gce_cert['client_email'],
-                'provider_content.certificate_path': settings.gce.cert_path,
+                'provider_content.json_key': json_key,
```

Comment:
@adarshdubey-star there are some more tests in ui/test_host that will require this update, are you going to update them as well? 
OR consider migrating them under ui/test_computeresource_gce as it make more sense to keep those here, wdyt?

Reply:
There is one in upgrades/test_host.py as well. Please update that test case too.

Reply:
I think we can migrate those test-cases in this module @Gauravtalreja1